common/test_local_filestore.py

```diff
@@ -47,6 +47,16 @@ def test_ls_non_must_exist(tmp_path):
     local_filestore.ls(str(file_path), must_exist=False)
 
 
+def test_ls_one_file_per_line(tmp_path):
+    """Tests ls will list files as one per line."""
+    dir_path = tmp_path
+    file1 = dir_path / 'file1'
+    file2 = dir_path / 'file2'
+    open(file1, "w+").close()
+    open(file2, "w+").close()
+    assert local_filestore.ls(str(dir_path)).output.count('\n') == 2
```

Comment:
why not compare to expected output, ie something like "file1\nfile2\n" ?

Reply:
+1

Reply:
May I know the reason why a complete output comparison is better?

Reply:
It's much more obvious what is being tested. The current version is a bit odd and implicit.

Reply:
(1) others will know exactly what the execution results should be.
(2) only use `\n` cannot distinguish the running results between `-l` and `-1`.